nntrainer/utils/util_func.h

```diff
@@ -271,6 +271,16 @@ template <typename T, typename C = int> T enum_class_or(T e1, T e2) {
   return static_cast<T>(i1 | i2);
 }
 
+/**
+ * @brief Convert a relative path into an absolute path.
+ *
+ * @param name relative path
+ * @param resolved variable to store the result value.
+ *
+ * @return absolute path
+ */
+char *getRealpath(const char *__restrict name, char *__restrict resolved);
```

Comment:
`__restrict` is optmization directive. Should we use this in here?
It doesnot needs to have disjoint property. 
please check: https://en.cppreference.com/w/c/language/restrict

Reply:
I didn't know about it. The PR has been modified. Thank you very much for letting me know.

Reply:
@jihochu I've removed `__restrict` in that function. Thank you :)